src/unistd.rs

```diff
@@ -3569,13 +3568,29 @@ impl Group {
     ///
     /// # Examples
     ///
+    // Disable this test on all OS except BSD as wheel group may not exist.
+    #[cfg_attr(not(any(apple_targets, bsd)), doc = " ```no_run")]
+    #[cfg_attr(any(apple_targets, bsd), doc = " ```")]
+    /// use nix::unistd::Group;
+    /// // Returns an Result<Option<Group>>, thus the double unwrap.
+    /// let group = Group::from_name("wheel").unwrap().unwrap();
+    /// assert!(group.name == "wheel");
+    /// let group_id = group.gid;
+    /// let group = Group::from_gid(group_id).unwrap().unwrap();
+    /// assert_eq!(group.gid, group_id);
+    /// assert_eq!(group.name, "wheel");
+    /// ```
     // Disable this test on all OS except Linux as root group may not exist.
     #[cfg_attr(not(target_os = "linux"), doc = " ```no_run")]
     #[cfg_attr(target_os = "linux", doc = " ```")]
     /// use nix::unistd::Group;
     /// // Returns an Result<Option<Group>>, thus the double unwrap.
-    /// let res = Group::from_name("root").unwrap().unwrap();
-    /// assert!(res.name == "root");
+    /// let group = Group::from_name("root").unwrap().unwrap();
+    /// assert!(group.name == "root");
+    /// let group_id = group.gid;
+    /// let group = Group::from_gid(group_id).unwrap().unwrap();
+    /// assert_eq!(group.gid, group_id);
+    /// assert_eq!(group.name, "root");
```

Comment:
And this change should be reverted as this example is only for the API `Group::from_name`, `from_gid` will be covered by its own example.

Reply:
done